### DISCUSSION TITLE
si possono inserire 2 indirizzi di segnalazione?

### DISCUSSION THREAD

**Discussion by Tascri** [globaleaks/globaleaks-whistleblowing-software#4129] [Q&A]
Avrei bisogno di capire come gestire una segnalazione nel caso in cui riguardi una delle persone che riceve, visualizza e gestisce le segnalazioni. Esiste la possibilità di inserire una domanda in cui si chiede se la segnalazione riguarda una delle persone destinate alla gestione della stessa e quindi, a risposta affermativa, l'invio viene fatto ad un  altro indirizzo mail?

Grazie

**Comment by evilaliv3** [comment]
Salve,

si è possibile.

Potete fare ad esempio una domanda a risposta multipla e associare un diverso ricevente in caso di una particolare opzione di risposta.

Provate a creare una domanda a scelta multipla, inserite delle opzion "SI/No" ad esempio e vedrete che in corrispondenza delle rispose trovate una icona a forma di ominino che vi permette di associare uno o piu riceventi a cui inviare la segnalazione in caso venga fatta tale scelta.
